ISSUE_TRIAGE.rst

```diff
@@ -75,6 +75,7 @@ Components
 - ``area/models``: MLmodel format, model serialization/deserialization, flavors
 - ``area/projects``: MLproject format, project execution backends
 - ``area/scoring``: Local serving, model deployment tools, Spark UDFs
+- ``area/server``: MLflow server, UI server
```

Comment:
@AveshCSingh What is the difference between `MLflow server` and `UI server`?

Reply:
`MLflow server` (started with `mlflow server`) is the Python server that serves requests (eg `/runs/create`), while the `UI server` (started with `mlflow ui`) serves the tracking UI. Is this the right distinction / clear to contributors?

Reply:
As discussed offline, the above is incorrect. `mlflow server` handles both API endpoints and serving the Javascript (via gunicorn), while the JS server is only used in development.

Thanks for your help understanding this! 

I've updated the PR to have a more clear area name, suggested by @sueann.